lib/index.js

```diff
@@ -22,14 +25,16 @@ AdWords.ensure(function(msg, settings){
   var os = msg.proxy('context.os') || {};
   var conversionId = settings.conversionId;
   var conversionLabel = mapper.getConversionLabel(msg.event(), settings.events);
+  var remarketing = settings.remarketing;
 
   if (!device.advertisingId) return this.invalid('All calls must have an advertisingId');
   if (!app.namespace) return this.invalid('All calls must have an App Namespace');
   if (!app.version) return this.invalid('All calls must have an App Version');
   if (!os.version) return this.invalid('All calls must have an OS Version');
   if (!device.type) return this.invalid('All calls must have a Device Type');
   if (!conversionId) return this.invalid('All calls must have an conversionId');
-  if (!conversionLabel) return this.invalid('All calls must have a conversionLabel');
+  // Remarketing tags do not require labels
+  if (!remarketing && !conversionLabel) return this.invalid('All calls must have a conversionLabel');
```

Comment:
It looks like the integration drops the message if there is no conversionLabel (even if its remarketing). If so we should probably still mark it invalid here.